package.json

```diff
@@ -60,5 +60,8 @@
     "dev-packages/*",
     "packages/*",
     "examples/*"
-  ]
-}
\ No newline at end of file
+  ],
```

Comment:
Please revert changes to this file, the root package should contain only dev dependencies